sdk/schemaregistry/azure-schemaregistry-avroencoder/migration_guide.md

```diff
@@ -0,0 +1,194 @@
+the benefits of doing so would be. As Azure has matured and been embraced by a more diverse group of developers,
+we have been focused on learning the patterns and practices to best support developer productivity and
+to understand the gaps that the Python client libraries have.
+
+There were several areas of consistent feedback expressed across the Azure client library ecosystem.
+One of the most important is that the client libraries for different Azure services have not had a
+consistent approach to organization, naming, and API structure. Additionally, many developers have felt
+that the learning curve was difficult, and the APIs did not offer a good, approachable,
+and consistent onboarding story for those learning Azure or exploring a specific Azure service.
+
+To try and improve the development experience across Azure services,
+a set of uniform [design guidelines](https://azure.github.io/azure-sdk/general_introduction.html) was created
+for all languages to drive a consistent experience with established API patterns for all services.
+A set of [Python-specific guidelines](https://azure.github.io/azure-sdk/python/guidelines/index.html) was also introduced to ensure
+that Python clients have a natural and idiomatic feel with respect to the Python ecosystem.
+Further details are available in the guidelines for those interested.
+
+### New Features
+
+We have a variety of new features in the version v1.0.x of the Schema Registry Avro Encoder library.
+
+- `AvroEncoder` sync and async classes provide the functionality to encode and decode content which follows a schema with the RecordSchema format, as defined by the Apache Avro specification. The Apache Avro library is used as the implementation for encoding and decoding.
+- Refer to the [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/schemaregistry/azure-schemaregistry-avroencoder/CHANGELOG.md) for more new features, changes, and bug fixes.
+
+## Important changes
+
+### The AvroEncoder
+
+The `AvroEncoder` has been renamed from the `AvroSerializer` and has the following methods:
+
+* `encode`, which replaces `serialize`
+* `decode`, which replaces `deserialize`
+
+#### Encode
+
+The positional parameter `value` has been renamed to `content`. If `value` parameter name was specified when calling `serialize`, it must be updated to `content`.
+
+Using the `AvroSerializer`:
+
+```python
+with serializer:
+    dict_data_ben = {"name": "Ben", "favorite_number": 7, "favorite_color": "red"}
+    payload_ben = serializer.serialize(value=dict_data_ben, schema=SCHEMA_STRING)
+    print(payload_ben)  # prints bytes b'<record format indicator><schema ID><Avro-encoded data>``
+```
+
+Using the `AvroEncoder`:
+
+```python
+with encoder:
+    dict_content = {"name": "Ben", "favorite_number": 7, "favorite_color": "red"}
+    message_content_ben = encoder.encode(content=dict_content, schema=definition)
```

Comment:
should this also be SCHEMA_STRING for consistency

Reply:
oo yep! will fix